src/python/grpcio/grpc/__init__.py

```diff
@@ -31,12 +31,33 @@
 
 import abc
 import enum
+import sys
 
 import six
 
 from grpc._cython import cygrpc as _cygrpc
 
 
+############################### Extension Shims ################################
```

Comment:
If you say yes to my unhappily-late changes on master, they should be done here too.